dex/mandos/calculate_rewards_for_given_position_after_compound.scen.json

```diff
@@ -47,14 +47,14 @@
 				"function": "calculateRewardsForGivenPosition",
 				"arguments": [
 					"100,050,000",
-					"0x000000041dcd6500000000000000000001000000000405f5e10000000002c3500000000405f6a450"
+					"0x000000041dcd65000000000000000000000000000000000a01000000000405f5e10000000002c3500000000405f6a450"
 				],
 				"gasLimit": "100,000,000",
 				"gasPrice": "0"
 			},
 			"expect": {
 				"out": [
-					"4999"
+					"4995"
```

Comment:
why this change ? does these changes resolve the compound problem ?

Reply:
it's the new penalty amount change. yes